api/endpoints/swipes.py

```diff
@@ -0,0 +1,79 @@
+from django.shortcuts import render
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.validators import ValidationError
+from django.http import HttpResponse
+from api.models import Task, User
+from api.common import *
+
+# Swipe-related endpoints reside here.
+
+# URI: /api/swipe/p/:taskid,volunteerid,match:
+# Expect:
+def pin_swipe(request, taskid, volunteerid, match):
+
+    if "uemail" not in request.session:
+        return httpBadRequest()
+
+    if not check_taskid(taskid):
+        return HttpResponse("Invalid taskid", status=400)
+    if not check_userid(volunteerid):
+        return HttpResponse("Invalid volunteerid", status=400)
```

Comment:
Hmm, so same deal as in messages.py; these seem too verbose. We should return httpBadRequest() and maybe use a logger here, instead: https://docs.djangoproject.com/en/3.0/topics/logging/
The front-end will not display these messages in the UI - at best, we'll display a message asking users to try again. Since these messages are meant to flag application bugs, I'm thinking they should be logs. But if you feel like they might be useful during integration, I'm OK with keeping them for now and moving to logs at a later date.